[2024-09-03T16:37:30.676+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-03T16:37:30.699+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_operator_dag_v1.get_num1 manual__2024-09-03T16:37:26.493519+00:00 [queued]>
[2024-09-03T16:37:30.707+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_operator_dag_v1.get_num1 manual__2024-09-03T16:37:26.493519+00:00 [queued]>
[2024-09-03T16:37:30.709+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 3
[2024-09-03T16:37:30.719+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_num1> on 2024-09-03 16:37:26.493519+00:00
[2024-09-03T16:37:30.730+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=17061) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-03T16:37:30.730+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'python_operator_dag_v1', 'get_num1', 'manual__2024-09-03T16:37:26.493519+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpnbdk578x']
[2024-09-03T16:37:30.733+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask get_num1
[2024-09-03T16:37:30.732+0000] {standard_task_runner.py:72} INFO - Started process 17063 to run task
[2024-09-03T16:37:30.782+0000] {task_command.py:467} INFO - Running <TaskInstance: python_operator_dag_v1.get_num1 manual__2024-09-03T16:37:26.493519+00:00 [running]> on host cedbab6b7256
[2024-09-03T16:37:30.872+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abhay' AIRFLOW_CTX_DAG_ID='python_operator_dag_v1' AIRFLOW_CTX_TASK_ID='get_num1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T16:37:26.493519+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T16:37:26.493519+00:00'
[2024-09-03T16:37:30.880+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-03T16:37:30.915+0000] {python.py:240} INFO - Done. Returned value was: 30
[2024-09-03T16:37:30.981+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-03T16:37:30.985+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=python_operator_dag_v1, task_id=get_num1, run_id=manual__2024-09-03T16:37:26.493519+00:00, execution_date=20240903T163726, start_date=20240903T163730, end_date=20240903T163730
[2024-09-03T16:37:31.031+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-03T16:37:31.057+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-03T16:37:31.064+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
